tools/modelgen/generators/templates/init.tmpl

```diff
@@ -0,0 +1,6 @@
+func Init() {
+  {{ range .Objects }} collections.{{ .Name }}s = make(map[string]*{{ initialCap .Name }})
+  {{ end }}
+  {{ range .Objects }} restore{{ initialCap .Name}}()
```

Comment:
may be some sort of error handling or logging would needed if `restore()` returns a failure. But that can be done outside this PR.